src/basics.md

```diff
@@ -502,6 +503,57 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-continuous-process-output]: #ex-continuous-process-output
+<a name="ex-continuous-process-output"></a>
+## Continuously process child process' outputs
+
+[![std-badge]][std] [![cat-os-badge]][cat-os]
+
+Contrary to [Run an external command and process
+stdout](#ex-parse-subprocess-output), while an external [`Command`] is
+running, process its standard output. A new pipe is created by
```

Comment:
the explanation is slightly unclear. I'd emphasize that the we do not wait for the child process to finish and process the output continuously.